deployment/ccip/changeset/solana/cs_chain_contracts.go

```diff
@@ -39,19 +38,20 @@ func GetTokenProgramID(programName string) (solana.PublicKey, error) {
 	return programID, nil
 }
 
+// TODO fix, token_pool gobinding was deleted and the contract seems to be different
```

Comment:
Can you pull in latest develop? This was fixed https://github.com/smartcontractkit/chainlink/pull/16291